### ISSUE TITLE
Sanguine infusion adding enchantments infinite-loops draining LP from the network

### ISSUE THREAD

**Issue by RonRicoSuave** [Arcaratus/BloodArsenal#145]
To repeat:  Create sword with ritual, about a 20 second delay with no indication of progress but items eventually consume and the sword is created.  

Now try to add looting / luck and watch the LP tick away.  The JEI page for the ritual says it requires 10k LP for this.  Without my mob spawners running to keep the network full, it drained a completely full network of LP over the course of a couple hours, and didn't consume any items or apply modifiers.  After wandering over to sacrifice and get rid of nausea effect, I shut down MC and restarted.  When I re-entered the world and headed over to the altar it had completed using the expected amount of LP and with the desired results.   

Tried again with a L5 modifer to the sword and it looked like an identical thing.  Let it drain 1M LP just to be safe and signed out.  Came back and checked.  It had consumed enough materials for one level of the modifier this time, left the rest, and was now draining MP constantly.  

There appears to be some state getting set that is only cleared on logout that prevents the ritual from completing when adding modifiers but not when creating the item in the first place.  

I also wasn't able to get the lightning sigil to do anything, and ended up cheating in stasis plates to see what was walled behind the crazy requirements, but that's not directly related.

**Comment by Arcaratus** [comment]
I will take a look into this for the next update.

**Comment by TrirdKing** [comment]
I wish I could even get to the point of adding modifiers to the stasis sword but because there is 0 documentation it's just a slightly better diamond sword for me

**Comment by rudger-wolvram** [comment]
I'm running into this as well (1.12.2 2.2.0-28). Created a stasis sword, tried to add Razor Sharp 6 and later 1 as a test. It just sat and ate 10m LP in about an hour without any modifier applying.

TiridKing, until the book can be updated, JEI is the only source I've found recipes in. You can click on the Stasis tools/blade and see what the ritual needs. The visual representation in JEI is off centered (e.g. showing items in 8 cardinal directions but the ritual is actually only 4 directions with 2 plates in each direction)

**Comment by rudger-wolvram** [comment]
So I did some more testing, it seems that a server reboot is needed between infusions. I tried making the stasis blade and did. Then tried to upgrade it with Razor sharp 6, ran into infinite drain loop. Rebooted server, tried again, infused it again and this time it worked. I now have +8 attack on the stasis handle... not the blade, the handle, so that's separate issue I'll be opening. But, point is, it does work, but seems to take a server reboot to kickstart it and it may only work once per reboot.
